package.json

```diff
@@ -30,12 +30,12 @@
   "license": "Apache-2.0",
   "dependencies": {},
   "engines": {
-    "node": "^8.0.0"
+    "node": ">=8.0.0"
   },
   "devDependencies": {
     "@types/node": "^10.12.19",
```

Comment:
if `@types/node` needs bumped to `>=10` too, just say the word.

Reply:
I think it would be safe to jump to >=12 since 12 is the lowest LTS version right now (see chart on https://nodejs.org/en/about/releases/)